src/_includes/banner.html

```diff
@@ -1,8 +1,3 @@
-{% if page.show_banner -%}
-  <!--{% assign fl_url = '/jobs/infrastructure' -%}-->
-  {% assign fl_url = '/create' -%}
-  <div class="site-banner site-banner--default" role="alert">
-    {% comment %} For headings use <h3> elements. {% endcomment -%}
-    <a href="/clock">See the winners of the Flutter Clock challenge!</a>
-  </div>
-{% endif -%}
+{% comment %}
+To get the banner back, mv oldbanner.html to banner.html
```

Comment:
Is there a reason not to just change show_banner to false? (That's how we usually do it on dart.dev.)